cmd/manager/main.go

```diff
@@ -194,14 +195,22 @@ func main() {
 		setupLog.Error(err, "unable to create CA certificate pool")
 		os.Exit(1)
 	}
-	unpacker := &source.ImageRegistry{
-		BaseCachePath:   filepath.Join(cachePath, "unpack"),
-		CertPoolWatcher: certPoolWatcher,
+
+	unpacker := &source.ContainersImageRegistry{
+		BaseCachePath: filepath.Join(cachePath, "unpack"),
+		SourceContext: &types.SystemContext{
+			DockerCertPath: caCertDir,
+			OCICertPath:    caCertDir,
+		},
+	}
+	defaultAuthConfigPath := filepath.Join("/etc", "containers", "auth.json")
+	if _, err := os.Stat(defaultAuthConfigPath); err == nil {
+		unpacker.SourceContext.AuthFilePath = defaultAuthConfigPath
```

Comment:
This doesn't seem to be a default for `containers/image`, interestingly. Researching this a bit more.

Reply:
Primary read/write file for auth seems to be `${XDG_RUNTIME_DIR}/containers/auth.json` or `$HOME/.config/containers/auth.json`: https://github.com/containers/image/blob/main/docs/containers-auth.json.5.md#description

Reply:
Looking at this, we should probably revisit the RFC and highlight the default location for all the configs we may care about

Reply:
So it turns out that file location is just not a thing (yet at least): https://github.com/containers/image/pull/1746

Reply:
We could follow the pattern that podman and skopeo do. If `REGISTRY_AUTH_FILE` is set in the environment, we set its value on this field. I'll switch the PR to do that for now.

Reply:
I took all of the auth stuff back out of the PR for now. We should discuss and handle separately.

This PR is now focused on `containers/image` and use of `/etc/containers` system configuration (as an unsupported/subject-to-change internal API)